_blog/why-null-models-crispr.md

```diff
@@ -0,0 +1,83 @@
+
+<p align="center">
+<img src="/img/posts/why-null-models-crispr/CRISPR_diagram_v3.svg" alt="crispr_diagram" width="600px" style="padding:5px;">
+</p>
+
+## Where do skewed spacer abundance distributions come from?
+
+Many different research groups have observed the same phenomenon: when bacteria use their CRISPR immune system, some spacers end up being a lot more common in the population than others. A *lot* more common: in the data we analyzed for this project, the most common spacer sequence was *ten thousand* times more common than the least common sequence. Distributions like this are often called ''long-tailed'' because in a histogram the data is smeared out for a long way on one or both sides: a long tail.
+
+This figure shows the abundance of each spacer sequence in the experiment, and each colour is a different day in the 14-day experiment. Each sequence is sorted from most common to least common (left to right), and the y-axis shows the frequency of each sequence. For most days, the most common sequence at rank 1 has a frequency of 10<sup>-1</sup> = 10%, and the least common sequences have frequencies around 10<sup>-5</sup>.
+
+<p align="center">
+<img src="/img/posts/why-null-models-crispr/banfield_data.svg" alt="banfield_data" width="400px" style="padding:5px;">
+</p>
+
+This study also found that certain regions of the phage genome were more likely to be a source for spacers, and combined with this skewed abundance distribution, they concluded that the high-abundance spacers were special: their sequence makes them more likely to be high abundance. This sounds like a perfectly reasonable explanation, and it's supported by other experiments. Here’s a figure from a another paper that looked specifically at the process of spacer acquisition - they sequenced new spacers 24 hours after bacteria were infected by phages without leaving the bacteria lots of time to grow and die. They performed the same experiment twice (two 'replicates'), and this plot shows the frequency of spacer sequences in the first replicate versus the frequency of the same spacer sequences in the second replicate. The data points are strongly correlated: they fall on or near a straight diagonal line with slope 1. This means that if a spacer sequence was highly abundant in the first experiment, it is also generally highly abundant in the second experiment, suggesting that there is indeed something about the spacer itself that destines it for high or low abundance[^3]. 
```

Comment:
How is the first part of the first sentence any different from the last paragraph? After finishing this paragraph I understood, but at first I was a little confused. 

For the second experiment, is this for individual bacteria or for two populations? 

Did the first study do anything to figure out that certain regions of the phage genome were more likely to be a source for spacers? Or did they just get to their conclusion from the skewed distribution? If it’s the latter, I think it would be cool if you waited to talk about the next study until after your null model. That way you could show how their conclusion might not be right, but that then with another experiment we can rule out the null model. You can conclude (like you do) by saying that experiments that can be reproduced like this are rare, and that often we only have the distributions. Otherwise it's a little back and forth: here's a skewed distribution -> here is the underlying reason behind this particular distribution -> but another one exists -> but it doesn't really matter because we all ready know what the real explanation is -> but in biology there are many systems where you couldn't do this second kind of experiment. Do you see what I mean? I think it would work better like: here's a skewed distribution -> conclusion from first study -> but wait, there's another explanation (enter null model) -> if we do more experiments we can rule out the null model -> more experiments often not possible.

Reply:
Yeah, I see what you're saying, but I think the point I was trying to make is a bit different, that you can't take "B causes A" to mean "A is evidence of B". I'll rework it to try to explain a bit better!